scripts/build/index.js

```diff
@@ -6,6 +6,7 @@ const {
   inputDir = join(process.cwd(), 'source'),
   outputDir = join(process.cwd(), 'dist'),
   basePath = '/',
+  assetPath = process.env.CDN_URL,
```

Comment:
I am also happy to make the env var ASSET_URL, I was just trying to meet Jon's request for 'I want to make it clear what we are setting this env var in config for'